setup/db/db/schema-452to460.sql

```diff
@@ -353,6 +353,8 @@ CREATE VIEW `cloud`.`user_vm_view` AS
         `cloud`.`user_vm_details` `custom_speed`  ON (((`custom_speed`.`vm_id` = `cloud`.`vm_instance`.`id`) and (`custom_speed`.`name` = 'CpuSpeed')))
            left join
         `cloud`.`user_vm_details` `custom_ram_size`  ON (((`custom_ram_size`.`vm_id` = `cloud`.`vm_instance`.`id`) and (`custom_ram_size`.`name` = 'memory')));
+        delete s1 from ssh_keypairs s1, ssh_keypairs s2 where s1.id > s2.id and s1.public_key = s2.public_key;
+        ALTER TABLE ssh_keypairs ADD UNIQUE (fingerprint);
```

Comment:
Here, we added unique constraint on fingerprint column, but in previous one you deleted the rows where public_key column values are same.

The bug mentions that "previous" ones added with same names are deleted, here it seems "id" is an auto  increment column and is deleting rows which are added "later", not the "previous" ones.